timed/subscription/tests/test_package.py

```diff
@@ -18,11 +19,15 @@ def test_subscription_package_list(auth_client):
 
 
 def test_subscription_package_filter_customer(auth_client):
-    other_project = ProjectFactory.create()
-    PackageFactory.create(billing_type=other_project.billing_type)
-
-    my_project = ProjectFactory.create()
-    package = PackageFactory.create(billing_type=my_project.billing_type)
+    customer = CustomerFactory.create()
+    billing_type = BillingTypeFactory.create()
+    package = PackageFactory.create(billing_type=billing_type)
+    ProjectFactory.create(billing_type=billing_type, customer=customer)
+
+    my_project = ProjectFactory.create(
```

Comment:
my_project var is not needed as below you could simply adjust it to user create customer

Reply:
the second project is to check that only one package is returned.